lib/jekyll/mime.types

```diff
@@ -363,24 +367,24 @@ application/vnd.ms-officetheme                                            thmx
 application/vnd.ms-outlook                                                msg
 application/vnd.ms-pki.seccat                                             cat
 application/vnd.ms-pki.stl                                                stl
-application/vnd.ms-powerpoint                                             ppt pps pot
+application/vnd.ms-powerpoint                                             pot pps ppt
 application/vnd.ms-powerpoint.addin.macroenabled.12                       ppam
 application/vnd.ms-powerpoint.presentation.macroenabled.12                pptm
 application/vnd.ms-powerpoint.slide.macroenabled.12                       sldm
 application/vnd.ms-powerpoint.slideshow.macroenabled.12                   ppsm
 application/vnd.ms-powerpoint.template.macroenabled.12                    potm
-application/vnd.ms-project                                                mpp mpt
+application/vnd.ms-project                                                mpt
```

Comment:
`.mpp` is being removed from the list.
Are we fine with this change in a minor version bump?

Reply:
I'm personally fine with it! We can always add it back in – I'd rather not over-optimize for backwards compatibility in this file for uncommon file types like `.mpp` (Microsoft Project file?).